tutorials/28_Structured_Output_With_OpenAI.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>let's try to always use "JSON"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/412/discussion/#comment-2581346695'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>